### PR TITLE
Replace OMDB with Spotify

### PR DESCRIPTION
**Pull request by tracymu**
Hey @puyo 

so I started updating the tutorial to use the Spotify API.

I did the first tutorial ok no problems.

With the second tutorial, I came across a problem with scaffolding/routes. This is nothing to do with the change in API - but a problem I have had with the routing.

I changed the routes to avoid the problem with the `show` page trying to render the `search` record when going to `favourites/search`. But then this stuffed up the saving of it to the database. 

I'm not sure if the problem is my reading comprehension skills ....

**Things I still have to do:**
- [ ] Update the images

### CODE REVIEW

File: _posts/2017-02-21-compare-favourite-songs.md [hunk 1]
```diff
@@ -61,15 +65,17 @@ Copy the search code from the last exercise into a new `search` method anywhere
 
     # Request info from API
     require 'net/http'
-    uri = URI.parse("http://www.omdbapi.com/?" + { s: q }.to_query)
+    uri = URI.parse("http://www.omdbapi.com/?" + { s: q, type: 'track' }.to_query)
```
Comment (puyo) [inline _posts/2017-02-21-compare-favourite-songs.md]: Still says OMDB?

Comment (tracymu) [reply _posts/2017-02-21-compare-favourite-songs.md]: whoops, must update, thanks

**Comment by tracymu** [comment]
Thanks so much @brentonannan  !!

**Comment by brentonannan** [comment]
Happy to help @tracymu!!

**Pull request merged**